docs/conf.py

```diff
@@ -55,7 +55,7 @@
 html_last_updated_fmt = ""
 
 html_theme_options = {
-    "repository_url": "https://github.com/sgl-project/sglang",
+    "repository_url": "https://github.com/sgl-project/sgl-project.github.io",
```

Comment:
It should not modify the generated file, as changes won't affect future outcomes. Instead, it should edit the source files at `https://github.com/sgl-project/sglang`.

Reply:
If users send PR to us in this repo, we can manually merge it in our main repo.